make_image.py

```diff
@@ -0,0 +1,186 @@
+from PIL import Image, ImageDraw, ImageFont, ImageColor
+import textwrap
+
+FONT = "Roboto-Regular.ttf"
+
+FONT_SIZE = 109
+BACKGROUND_COLOR = (0, 0, 0)
+COLOR = (255, 255, 255)
+WRAP_WIDTH = 30
+PADDING = 10
+MARGIN = 15
+
+
+def make_image(photo: str, quote: str, name: str, result:  str, context: str = ""):
+
+    with Image.open(photo) as img:
+        quote = '“{}”'.format(quote)
+        img = img.convert("L")
+        img_caption = text_image(quote)
+        img_author = text_image(
+            name, font_size=int(FONT_SIZE * 0.5), padding=20)
+        img_context = text_image(
```

Comment:
E quando for `/quote` sem contexto? Talvez precise de um condicional aqui.

Reply:
Recebo o erro `caused error "'NoneType' object has no attribute 'expandtabs'` ao rodar um `/quote`, acredito que esteja relacionado.